openshift/templates/partition_filler_job.yaml

```diff
@@ -0,0 +1,85 @@
+kind: "Template"
+apiVersion: "template.openshift.io/v1"
+metadata:
+  name: partition-filler-job-template
+  annotations:
+    description: "Job to fill partitioned tables with data from original table."
+    tags: "job,sfms"
+labels:
+  app.kubernetes.io/part-of: "${NAME}"
+  app: ${NAME}-${SUFFIX}
+parameters:
+  - name: NAME
+    description: Module name
+    value: wps
+  - name: SUFFIX
+    description: Deployment suffix, e.g. pr-###
+    required: true
+  - name: GLOBAL_NAME
+    description: Name of global Module
+    value: wps-global
+  - name: PROJ_TOOLS
+    value: e1e498-tools
+  - name: "TAG_NAME"
```

Comment:
Are the quotes needed?

Reply:
I don't see why they would be, I removed them.